.circleci/config.yml.in

```diff
@@ -359,6 +359,33 @@ jobs:
       - run_tests_selective:
           file_or_dir: test/test_prototype_*.py
 
+  unittest_jpeg_ref:
+    docker:
+      - image: condaforge/mambaforge
```

Comment:
Any chance we could stick to conda instead of mamba, so as to stick to the current tools that we use in the rest of the job?

Reply:
We probably can (there is most likely a docker image that gives us `conda`), but I'm not sure we should. `mamba` is a re-implementation of the protocol, but faster. So we effectively waste CI resources by using `conda`.

Reply:
Shouldn't we rely on the same image as the other `unittest` jobs here, to avoid any risk of deviating from our "baseline" testing infra? I believe they all rely on `conda` as well so there must be a way to install conda, even if we don't choose `condaforge/mambaforge` as the image

Reply:
Besides our CPU workflows on Linux, none of the other workflows even use `docker`:

https://github.com/pytorch/vision/blob/d675c0c6e94ee267b31e29a46d05421e14bf3370/.circleci/config.yml#L719-L722

https://github.com/pytorch/vision/blob/d675c0c6e94ee267b31e29a46d05421e14bf3370/.circleci/config.yml#L758-L761

https://github.com/pytorch/vision/blob/d675c0c6e94ee267b31e29a46d05421e14bf3370/.circleci/config.yml#L808-L811

https://github.com/pytorch/vision/blob/d675c0c6e94ee267b31e29a46d05421e14bf3370/.circleci/config.yml#L846-L849

https://github.com/pytorch/vision/blob/d675c0c6e94ee267b31e29a46d05421e14bf3370/.circleci/config.yml#L893-L896

Apart from that, they all download the `miniconda` installer:

https://github.com/pytorch/vision/blob/d675c0c6e94ee267b31e29a46d05421e14bf3370/.circleci/unittest/linux/scripts/setup_env.sh#L25

https://github.com/pytorch/vision/blob/d675c0c6e94ee267b31e29a46d05421e14bf3370/.circleci/unittest/windows/scripts/setup_env.sh#L22

There is a [container](https://hub.docker.com/r/conda/miniconda3) for that, but again, I see no point. It makes no difference if you install the environment with `conda` or `mamba` other than the latter is significantly faster.

Reply:
I'm happy to switch all of our jobs to mamba in another PR, but could we please stick to the same workflow that the other jobs are using? There is value in avoiding discrepancies with the rest of our CI jobs.

Reply:
What other workflow? Manually installing `conda` and setting things up or can I at least use a docker image with `conda` pre-installed?

Reply:
Sorry I'm a bit confused: what prevents us from staying as close as possible to `  unittest_linux_cpu:` ?